UefiPayloadPkg/Include/Library/DxeHobListLib.h

```diff
@@ -0,0 +1,27 @@
+/** @file

```

Comment:
should we change the name of DxeHobListLib?  Payload entry will also linke it. @niruiyu

Reply:
there are two HobList in UefiPayloadPkg: DxeHobLib and PayloadEntryHobLib. The latter one is linked to PayloadEntry.